src/components/NavItems/Assistant/AssistantScrapeResults/AssistantLinkResult.jsx

```diff
@@ -2,24 +2,116 @@ import React from "react";
 import Typography from "@mui/material/Typography";
+import CheckCircleOutlineIcon from "@mui/icons-material/CheckCircleOutline";
+import ErrorOutlineOutlinedIcon from "@mui/icons-material/ErrorOutlineOutlined";
+import SentimentSatisfied from "@mui/icons-material/SentimentSatisfied";
 import { i18nLoadNamespace } from "components/Shared/Languages/i18nLoadNamespace";
 import useMyStyles from "../../../Shared/MaterialUiStyles/useMyStyles";
+import ExtractedSourceCredibilityResult from "../AssistantCheckResults/ExtractedSourceCredibilityResult";
+import Tooltip from "@mui/material/Tooltip";
+import HelpOutlineOutlinedIcon from "@mui/icons-material/HelpOutlineOutlined";
+
+const ExtractedUrl = (index, keyword, extractedSourceCred, link) => {
+  let sourceType;
+  let Icon;
+  let iconColor;
+
+  if (extractedSourceCred) {
+    if (extractedSourceCred[link].caution) {
+      sourceType = keyword("warning");
+      Icon = ErrorOutlineOutlinedIcon;
+      iconColor = "error";
+    } else if (extractedSourceCred[link].mixed) {
+      sourceType = keyword("mentions");
+      Icon = SentimentSatisfied;
+      iconColor = "action";
+    } else if (extractedSourceCred[link].positive) {
+      sourceType = keyword("fact_checker");
+      Icon = CheckCircleOutlineIcon;
+      iconColor = "primary";
+    }
+  }
+  return (
+    <Grid container wrap="wrap" key={index}>
+      <Grid item xs={1} align="center">
+        <LinkIcon />
+      </Grid>
+
+      <Grid item xs={10} align="left">
+        <Typography>
+          <Link
+            color={
+              extractedSourceCred
+                ? extractedSourceCred[link].urlColor
+                : "inherit"
+            }
+            href={
+              extractedSourceCred
+                ? extractedSourceCred[link].resolvedLink
+                : link
+            }
+          >
+            {extractedSourceCred
+              ? extractedSourceCred[link].resolvedLink
+              : link}
+          </Link>
+        </Typography>
+      </Grid>
+
+      {sourceType ? (
+        <Grid item xs={1} style={{ display: "flex", alignItems: "center" }}>
+          <ExtractedSourceCredibilityResult
+            extractedSourceCredibilityResults={extractedSourceCred[link]}
+            sourceType={sourceType}
+            Icon={Icon}
+            iconColor={iconColor}
+            url={extractedSourceCred[link].resolvedLink}
+            urlColor={extractedSourceCred[link].urlColor}
+          />
+        </Grid>
+      ) : null}
+    </Grid>
+  );
+};
+
+const ExtractedUrlList = (
+  keyword,
+  linkList,
+  extractedLinks,
+  extractedSourceCred,
+) => {
+  const links = extractedLinks ? extractedLinks : linkList ? linkList : null;
+  return (
+    <div>
+      {links ? (
+        links.map((link, index) =>
+          ExtractedUrl(index, keyword, extractedSourceCred, link),
+        )
+      ) : (
+        <p>Failed to load extracted URLs with URL Domain Analysis. </p>
```

Comment:
Needs i18n.

Reply:
I could use this one:
`sc_failed  The source credibility check has failed. Some results may not be displayed. If the problem persists, please contact support.`
or should I write a new one as above?

Reply:
Writing a new one - awaiting translation checks.